data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/lightclient/GetLightClientUpdatesByRange.java

```diff
@@ -71,6 +79,17 @@ public void handleRequest(RestApiRequest request) throws JsonProcessingException
                 .withField("data", lightClientUpdateType, ObjectAndMetaData::getData)
                 .build();
 
-    return SerializableTypeDefinition.listOf(lightClientUpdateObjectType);
+    return new JsonResponseContentTypeDefinition<>(
+        SerializableTypeDefinition.listOf(lightClientUpdateObjectType));
+  }
+
+  private static ResponseContentTypeDefinition<List<LightClientUpdateResponse>>
+      getSszResponseType() {
+    OctetStreamResponseContentTypeDefinition.OctetStreamSerializer<List<LightClientUpdateResponse>>
+        serializer =
+            (data, out) ->
+                data.stream().forEachOrdered(lcuResponse -> lcuResponse.sszSerialize(out));
```

Comment:
I had previously used `forEach` until I read that one does not guarantee ordering... I hope that collection streaming plays well with serialization, if not I am open to suggestions.

Reply:
I wonder if maybe we add this data type to `prettyPrint` so that we can see the output in json?

`PrettyPrintCommand`

Reply:
That's a neat idea. How do we add it to `PrettyPrintCommand`? Is it sufficient to implement `getJsonTypeDefinition()` in `LightClientUpdateResponseSchema`?

Reply:
Not sure to be honest for a non spec type, one of us will have to play and see what happens, you use it via debugTools
I know for SignedBeaconBlock I can just do
```
 build/install/teku/bin/teku debug-tools pp capella SignedBeaconBlock a.ssz
 ```
 
 and i think it'd be fair to expect something similar where we specify the type and it can read from the SSZ file and output the json.  The typeDefinition is the way to go for that bit, just a matter of whether we need to do some handling for the type bit in the command itself...

I'm off-site next week but can probably play after that, I might get a little extra time, but its work trip so might be nuts...
If you like, have a play and see what you think...